src/Linux/Avalonia.LinuxFramebuffer/Input/LibInput/LibInputKeyCodes.cs

```diff
@@ -0,0 +1,588 @@
+// ReSharper disable UnusedMember.Global
+// ReSharper disable InconsistentNaming
+// ReSharper disable IdentifierTypo
+// ReSharper disable CommentTypo
+namespace Avalonia.LinuxFramebuffer.Input.LibInput
+{
+    public enum LibInputKeyCodes : uint
```

Comment:
Those are actually evdev codes. I have a generator script for getting those from system headers - https://github.com/kekekeks/evdev-sharp/blob/master/src/EvDevSharp/gencodes.cs.script

Pregenerated file - https://github.com/kekekeks/evdev-sharp/blob/master/src/EvDevSharp/Codes.generated.cs

Reply:
Ah sweet, I was looking for something like that. Should we take your library in as a dependency or just use the script, if we do how will that affect CI?

Reply:
The library is by no means complete, I was exploring alternatives before taking dependency on libinput.